app/src/main/java/com/cryart/sabbathschool/view/SSReadingView.java

```diff
@@ -320,6 +316,20 @@ public void onPageFinished(WebView view, String url) {
             updateHighlights();
             updateComments();
         }
+
+        private void openExternalLink(Context context, String url) {
+            CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder()
+                    .setShowTitle(true)
+                    .enableUrlBarHiding()
+                    .setStartAnimations(context, R.anim.slide_up, android.R.anim.fade_out)
+                    .setExitAnimations(context, android.R.anim.fade_in, R.anim.slide_down);
+            try {
+                CustomTabsIntent intent = builder.build();
+                intent.launchUrl(context, Uri.parse(url));
+            } catch (Exception ex) {
+                Timber.e(ex);
+            }
```

Comment:
Is there a specific reason to use Custom Tabs? Do you think its better than FinestWebView?